_posts/2015-02-18-ApiBlueprint-button.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Social blueprint
+excerpt: Share your blueprint with others and learn from masters.
+layout: post
+date: 2015-02-18 00:00:00 +0100
+author: vincenzo
+published: true
+commets: true
+---
+
+We have recently received a lot of feedback from users that would like to learn more about API Blueprint and how to present stunning and beautiful documentation.
+
+We care about your feedback, and we think that documentation and tutorials about [API Blueprint](http://apiblueprint.org) are extremely important.
+
+One of the most requested features was to have the ability to look to the other blueprints as a source of inspiration, and today this is possible for all public API Blueprints.
+
+Whenever you're surfing a public blueprint, a new **API Blueprint** button will appear in header bar. Clicking this button will open up a new window with readable markdown, that can be easily grabbed, studied, modified and republished.
```

Comment:
Does the API Blueprint 'clone' the blueprint locally? If so, here's a suggestion: 'Similar to Github's 'Clone' feature, when you find an API blueprint that you'd like to reference or contribute to, you can easily clone that blueprint by clicking the API Blueprint button found within the navigation header.'